test/chrono-test.cc

```diff
@@ -879,4 +879,20 @@ TEST(chrono_test, timestamps_sub_seconds) {
       t10(std::chrono::milliseconds(2000));
 
   EXPECT_EQ(fmt::format("{:%S}", t10), "02.000");
+
+  {
+    const auto epoch = std::chrono::time_point<std::chrono::system_clock,
+                                               std::chrono::milliseconds>();
+    const auto d = std::chrono::milliseconds(250);
+
+    EXPECT_EQ("59.000", fmt::format("{:%S}", epoch - 4 * d));
+    EXPECT_EQ("59.250", fmt::format("{:%S}", epoch - 3 * d));
+    EXPECT_EQ("59.500", fmt::format("{:%S}", epoch - 2 * d));
+    EXPECT_EQ("59.750", fmt::format("{:%S}", epoch - 1 * d));
```

Comment:
I'm not sure if we need all four of these checks because they are effectively testing the same thing. One is probably enough.